_posts/2021-07-20-5.0.0-rs-release.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: Apache Arrow Rust 5.0.0 Release
+date: "2021-07-20 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+We recently released the 5.0.0 Rust version of [Apache Arrow](https://arrow.apache.org/), and this post highlights some of the improvements we have made. The full changelog can be found [here](https://github.com/apache/arrow-rs/blob/5.0.0/CHANGELOG.md).
+
+<!--
+(arrow_dev) alamb@MacBook-Pro:~/Software/arrow-rs$ git log --pretty=oneline 4.0.0..5.0.0 | wc -l
+     161
+(arrow_dev) alamb@MacBook-Pro:~/Software/arrow-rs$ git shortlog -sn 4.0.0..5.0.0 | wc -l
+      35 // but Jorge is double counted
+-->
+
+The Rust Arrow implementation would not be possible without the wonderful work and support of our community, and the 5.0.0 release is no exception. It includes 161 commits from 34 individual contributors, many of them with their first contribution. Thank you all very much.
+
+# Arrow
+This release contains bug fixes, performance improvements, and a slimmer dependency stack when default features are disabled. Documentation has been improved and more examples have been added.
```

Comment:
Pet peeve: every release announcement everywhere says "bug fixes and performance improvements" to the point of it being meaningless. IMO it's better to call out specific bugs/perf improvements if they are widely noticeable, otherwise don't bother saying this; if anyone wants to know specifically what bugs were fixed, they can read the changelog.

Reply:
Reworded in 9569e05357822728fef9bbb05fa4462f439fb5f8